VMBackup/main/ScriptRunner.py

```diff
@@ -81,16 +81,25 @@ def get_config(self):
         try:
             with open(self.configLocation, 'r') as configFile:
                 configData = json.load(configFile)
```

Comment:
In python,variables are scoped to innermost method it belongs to.if/for/with blocks doesn't count . So scoping is not issue but in our case if we don't enter the "with" block and access the configData, it throws error which is caught in the exception handling.